### ISSUE TITLE
.weights

### ISSUE THREAD

**Issue by evertonesm** [sowmyadvn/Crime-Detection--using-Deep-learning#3]
Sorry but is the weigths files available to test the detection?

**Comment by moeiscool** [comment]
i am also looking for it :( i went as far as to message the dev on Quora, hopefully I'll get a response

**Comment by bibinkt** [comment]
Hey guys,

We can develop the wieght by our self . but its littil bit of manual effort
are there , if you are interested we can work together and get it done !!

**Comment by moeiscool** [comment]
ive started developing one this morning based on this tutorial
https://github.com/spmallick/learnopencv/tree/master/YOLOv3-Training-Snowman-Detector

read over all the steps and before you start modify this file
https://github.com/spmallick/learnopencv/blob/master/YOLOv3-Training-Snowman-Detector/getDataFromOpenImages_snowman.py

change the `classes` to an array with your desired thingies. usable class names i found here 
https://storage.googleapis.com/openimages/2018_04/class-descriptions-boxable.csv

I'm doing this.
```
["Handgun","Rifle","Shotgun","Knife","Sword","Weapon","Screwdriver","Baseball bat","Flashlight","Dagger","Hammer","Torch","Person"]
```

im still in the downloading phase because "Person" has over 100,000 images 🤦‍♂️ 


I know this tutorial works because I followed it exactly as it was and made a snowman detector. So I expect it will do the same for my current selection of classes 🤞

**Comment by bibinkt** [comment]
Gr8 buddy .. you are awesome :-) let me try as well.. i was trying to pull
the image from google and else where :-(

**Comment by myounus96** [comment]
Why you wanna train the model yourself when you get [yolov3 model](https://github.com/AlexeyAB/darknet#pre-trained-models) (check the first one) trained on [openImages dataset](https://storage.googleapis.com/openimages/web/index.html) .openImages have 'Person' class. @moeiscool 
And @bibinkt  you can also use that because openImages also have classes related to weapons.

**Comment by bibinkt** [comment]
Yeah I tested that model but those are not effective but it works fine for person and other objects

**Comment by myounus96** [comment]
yup for some input videos it is unable to detect weapons,so did you came up with something better,can you please share?Thanks
@bibinkt
